.github/workflows/example.yml

```diff
@@ -44,10 +51,11 @@ jobs:
           stack-arguments: --resolver ${{ matrix.resolver }}
           cache-prefix: ${{ matrix.resolver }}/
 
-      - if: ${{ runner.os != 'windows' }}
+      - if:
+        shell: bash
```

Comment:
```suggestion
      - shell: bash
```

Reply:
done in https://github.com/freckle/stack-action/pull/35/commits/029a410fb4fa1d794463eeb2ad90493cd815f38f